build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1743,6 +1750,63 @@ message DigestFunction {
     // cryptographic hash function and its collision properties are not strongly guaranteed.
     // See https://github.com/aappleby/smhasher/wiki/MurmurHash3 .
     MURMUR3 = 7;
+
+    // The SHA-256 digest function, modified to use a Merkle tree for
+    // large objects. SHA256TREE hashes are computed as follows:
+    //
+    // - For blobs that are 1024 bytes or smaller, the hash is computed
+    //   using the regular SHA-256 digest function.
+    //
+    // - For blobs that are more than 1024 bytes in size, the hash is
+    //   computed as follows:
+    //
+    //   1. The blob is partitioned into a left (leading) and right
+    //      (trailing) blob. These blobs have lengths m and n
+    //      respectively, where m = 2^k and 0 < n <= m.
```

Comment:
There is only one `k` for which `len(blob) = m + n && 0 < n <= m` is satisfiable.

Reply:
Do you want to just change the letter for this, to avoid confusion with the `k` used below for the `chunk_size`. L1874.
As point 2. will repeat chunking until this `k'` is that `k`.

I think we can also clear up `k`'s relation to `1024` above this algorithm description. Maybe move the high-level explanation L1870-L1873 before this. And maybe add a short mention that `k` can well be 10. Then the three mentions would be more consistent